context/app/static/js/components/Routes/useSendPageView.js

```diff
@@ -8,6 +8,11 @@ function useSendPageView(path) {
       ReactGA.pageview(pathWithoutUUID);
       return;
     }
+    // send path without ID for specific saved list page
+    if (path.match(/\/my-lists\/[\w-]+/)) {
```

Comment:
```suggestion
    if (path.startswith('/my-lists/') { // Distinguished by final slash. 
```
Avoid regexes if you can.

Reply:
@mccalluc Well, I'm specifically looking for /my-lists/{uuid} where uuid is a dash separated string since just /my-lists is the lists page.